src/components/global/FormFieldTextRequired.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+/**
+ * FormFieldTextRequired Component
+ *
+ * The `FormFieldTextRequired` displays text required input.
+ *
+ * @description * Use this component to render text required input in forms.
+ *
+ * Note: This component is commonly used in `FormRegisterCoordinator`, `FormRegisterChallenge`.
+ *
+ * @props
+ * - `value` (string, required): The object representing user input.
```

Comment:
`value` prop is valid component prop?

Reply:
No, correct prop is `modelValue`.
Fixed.